bin/verify-testgrid-run.sh

```diff
@@ -0,0 +1,45 @@
+#!/bin/bash
+
+set -euo pipefail
+
+function log() {
+    echo "$1" 1>&2
+}
+
+function bail() {
+    log "$1"
+    exit 1
+}
+
+function main() {
```

Comment:
`curl -s https://api.testgrid.kurl.sh/api/v1/runs?searchRef=` with an empty STAGING_RELEASE returns results. should you protect against an empty variable STAGING_RELEASE?

Reply:
The job that calls this script (`verify-staging-release-testgrid-result`) depends on the job `get-staging-release` which should ensure the `STAGING_RELEASE` envar is set. I'll add a `require()` for the envar as a safeguard.

Reply:
thanks